cli/src/main.rs

```diff
@@ -1,24 +1,30 @@
-//! Iroha peer command line
+//! Iroha peer command-line interface.
 
 use std::str::FromStr;
 
-use color_eyre::Report;
-use iroha_core::{prelude::AllowAll, Arguments, Iroha};
+use iroha::Arguments;
+use iroha_core::prelude::AllowAll;
 use iroha_permissions_validators::public_blockchain::default_permissions;
 
 #[tokio::main]
-async fn main() -> Result<(), Report> {
+async fn main() -> Result<(), color_eyre::Report> {
     let mut args = Arguments::default();
-
-    if std::env::args().any(|arg| arg == "--help" || arg == "-h") {
+    if std::env::args().any(|a| is_help(&a)) {
         print_help();
         return Ok(());
     }
 
-    if std::env::args().any(|arg| arg == "--submit" || arg == "--submit-genesis" || arg == "-s") {
+    if std::env::args().any(|a| is_submit(&a)) {
         args.submit_genesis = true;
     }
 
+    for arg in std::env::args().skip(1) {
+        if !is_help(&arg) && !is_submit(&arg) {
```

Comment:
> Cause it can be painful in future if we will have a lot of complex flags.

We won't. If we do, we'll revert back to using `Clap`. The whole point is not to import a large crate for two flags.